Short note to motivate probabilistic scoring (#678)
nfidd/nfidd: sessions/forecast-evaluation.qmd

# Introduction to forecast evaluation

An important aspect of making forecasts is that we can later confront the forecasts with what really happened and use this to assess whether our forecast model makes good predictions, or which of multiple models work best in which situation.

> One option is to evaluate a single point prediction against a single observation: error = point prediction - observation.

Reviewer: ```suggestion
One option is to evaluate a single point prediction against a single observation:

$$
\text{error} = \text{prediction} - \text{observation}
$$
```

> You may have seen these kinds of metrics: for example, mean absolute error, percentage error, root mean square error...

Reviewer: ```suggestion
You may have seen these kinds of metrics: such as mean absolute error, mean percentage error, or root mean square error.
```
